src/udiskslinuxencrypted.c

```diff
@@ -272,9 +283,18 @@ handle_unlock (UDisksEncrypted        *encrypted,
   gchar *crypttab_passphrase = NULL;
   gchar *crypttab_options = NULL;
   gchar *device = NULL;
+  gchar *old_hint_encryption_type;
   gboolean read_only = FALSE;
+  gboolean hidden = FALSE;
+  gboolean system = FALSE;
```

Comment:
Please use ``is_system`` or something to avoid hiding the ``system()`` function.

Reply:
Done in e4d8eb277f98ca060b609d9cbf363c48ffa97702.